config/intellij-idea-inspections.xml

```diff
@@ -2149,6 +2148,8 @@ isolated classes and we cannot put them to separate package as it will affect us
                 <option value="ClassIndependentOfModule" />
                 <!-- till #4870, used in MT check markers -->
                 <option value="unused" />
+                <!-- we do not control the extended class and we don't want to create our own -->
+                <option value="AbstractClassExtendsConcreteClass" />
```

Comment:
we allow this suppression only till https://github.com/checkstyle/checkstyle/issues/5338 as it is used only for AbstractLoader.
ones this issue fixed, this allowance should also be removed.
Please update comment.

Reply:
@romani #5338 will not do anything with the suppression. The suppression is because `AbstractLoader` extends a concrete class. 
#5338 is just renaming and moving the class. It has nothing to do with it's inheritance and so suppression is still needed once #5338 .

The only way to remove this suppression is to rework the inheritance and implement our own `EntityResolver` but you didn't want that. See https://github.com/checkstyle/checkstyle/pull/5322#issuecomment-349348936 and https://github.com/checkstyle/checkstyle/pull/5322#pullrequestreview-82182260

Reply:
>is just renaming and moving the class

Does it make sense to make it non abstract in same #5338 issue ? Just remove abstract modifier.

Reply:
@romani We can't make it concrete as 4 other classes extend it.
`InternalLoader` (part of `ConfigurationLoader`), `PackageNamesLoader`, `ImportControlLoader` and `SuppressionsLoader`.
It would be the same problem over again.

IMO, we should make our own `EntityResolver` instead of using `DefaultHandler`.

Reply:
Nevermind my other point, i forgot one class must be abstract.

We can make it non-abstract, I don't see a problem with that.

Reply:
Updated issue and updated PR.